gitops/snapshot.go

```diff
@@ -344,6 +344,7 @@ func HaveAppStudioTestsSucceeded(snapshot *applicationapiv1alpha1.Snapshot) bool
 	if meta.FindStatusCondition(snapshot.Status.Conditions, AppStudioTestSucceededCondition) == nil {
 		return meta.IsStatusConditionTrue(snapshot.Status.Conditions, LegacyTestSucceededCondition)
 	}
+
```

Comment:
Why was this newline added?

Reply:
i ran go fmt ./... if it was added automatically it will add again after I remove it. But I will try remove it and format it again and see what happens